docker/ubuntu18/Makefile

```diff
@@ -0,0 +1,22 @@
+OS = ubuntu18
+TARBALL = cbmc.tar.gz
+
+USER = `whoami`
+GROUP = `groups | awk '{print $$1}'`
+
+default: container
+
+container:
+	docker build --tag cbmc-$(OS) .
+
+tarball:
+	docker run -v `pwd`:/home cbmc-$(OS) cp /tmp/$(TARBALL) /home
```

Comment:
Should this depend on `container?`

Reply:
I'm reluctant to have one phony target depend on another.  If the container target built something that could be tested, I'd be more inclined.